tests/tenant_test.go

```diff
@@ -50,12 +50,12 @@ func TestTenant(t *testing.T) {
 	var stagingInstanceDirs []string
 	var prodInstanceDirs []string
 	for i := 0; i < stagingTenantNumber; i++ {
-		instanceDir, err := ctl.provisionSQLiteInstance(instanceRootDir, fmt.Sprintf("%s-%d", stagingInstanceName, i))
+		instanceDir, err := provisionSQLiteInstance(instanceRootDir, fmt.Sprintf("%s-%d", stagingInstanceName, i))
 		a.NoError(err)
 		stagingInstanceDirs = append(stagingInstanceDirs, instanceDir)
 	}
 	for i := 0; i < prodTenantNumber; i++ {
-		instanceDir, err := ctl.provisionSQLiteInstance(instanceRootDir, fmt.Sprintf("%s-%d", prodInstanceName, i))
+		instanceDir, err := provisionSQLiteInstance(instanceRootDir, fmt.Sprintf("%s-%d", prodInstanceName, i))
```

Comment:
We may want to keep the ctl scope so that we know provisionSQLiteInstance() is part of controller class instead of tests package.

Reply:
SG.